patternfly/patternfly-elements
# tools: Add labels to PR templates, remove issue header

## Add support for auto-labeling and tidying templates for PRs

### What has changed and why

Summarize files edited as part of this MR along with a brief description of what was changed/why.

- Update PR templates to remove the issue boilerplate b/c PR templates work differently
- Update open PR script to parse labels out of template and add them to the query string

### Testing instructions

On this branch, run `npm run pr`:

```
? Choose your template: new component
? Additional information? No
```

Should open URL: `https://github.com/patternfly/patternfly-elements/compare/master...feat-pr-labels?template=new_component.md&labels=feature,needs:%20branch%20testing,needs:%20browser%20testing,needs:%20code%20review,priority:%20low`

Should apply the following labels:
<img width="372" alt="Screen Shot 2020-09-11 at 2 03 38 PM" src="https://user-images.githubusercontent.com/1840295/92958082-9e092700-f437-11ea-8d02-017fae1bcba3.png">

(It will pick up the old templates because those are what's on master.)

### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable to your PR.


### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
